modules/packages/URL.chpl

```diff
@@ -51,7 +51,7 @@ module URL {
                      start:int(64) = 0, end:int(64) = max(int(64)),
                      style:iostyle)
                     : fileReader(kind, locking) throws {
-    return openUrlReaderHelper(url, kind, locking, start, end,
+    return openUrlReaderHelper(url, kind, locking, start..end,
```

Comment:
Based on the documentation for the function and missing this myself when I was updating the IO functions, I believe this should actually be `start..(end-1)` when `end != max(int(64))` (and either what you have here or `start..` when `end == max(int(64))`).  That's because end had been the first index that should *not* be included.

This will apply to most of the conversions in the file, unfortunately